clustered/clustered-dist/build.gradle

```diff
@@ -79,7 +79,7 @@ jar {
     'Bundle-SymbolicName': 'org.ehcache.clustered',
     'Export-Package': '!com.tc.*, !com.terracotta.*, !org.terracotta.*, !org.ehcache.*.internal.*, !sun.misc, ' +
       'org.ehcache.clustered.client.*, org.ehcache.clustered.common.*',
-    'Import-Package': '!sun.misc.*, org.ehcache.xml.*;resolution:=optional, *'
+    'Import-Package': '!sun.misc.*, org.ehcache.xml.*;resolution:=optional, jdk.jfr.*;resolution:=optional, *'
```

Comment:
This makes sense. Have you confirmed however that this stuff does work correctly if the bundle is available?

Reply:
I've confirmed everything works if everything is present and a tripwire appender is added to logback.